### ISSUE TITLE
Open Data Package Error on Windows only

### ISSUE THREAD

**Issue by louisjasek** [ODIQueensland/data-curator#804]
### Current Behaviour (for problems)
> When I open this data package (https://github.com/frictionlessdata/example-data-packages/raw/master/zip/cpi.zip) from URL on Windows, it spits out an error message. This does not occur when I open the same data package from URL on mac.

![open data package error](https://user-images.githubusercontent.com/36529635/40458459-681e4678-5f3f-11e8-8f66-b34a35a3be2a.PNG)

### Expected Behaviour
> When opening this data package, no error should occur and the data package should open correctly

### Steps to Reproduce

1. Open Data Curator using the icon
2. Click File > Open Data Package > Zip from URL
3.Enter this URL: https://github.com/frictionlessdata/example-data-packages/raw/master/zip/cpi.zip
4. Click Open URL
5. Error appears.

### Your Environment

Data Curator version: 0.17.0
Operating System and version: Windows 10 Pro

**Comment by louisjasek** [comment]
@Stephen-Gates if you wouldn't mind adding priority and milestone to this one too. Again I feel this is high priority given it is against core functionality. It seems to be loading some of the package (e.g. the data) but not all of it. If it was just an error message but everything worked it wouldn't be as high priority. Keen on your thoughts though.

**Comment by louisjasek** [comment]
This is occurring when opening data package from file as well. For example if you were to download that file to your desktop and then open from file from data curator.

**Comment by Stephen-Gates** [comment]
Explore upgrading to the latest version of HandsOnTable

**Comment by mattRedBox** [comment]
nothing to do with handsontable - think that comment is meant for #805

**Comment by mattRedBox** [comment]
error in path creation - will make this os-dependent.

**Comment by mattRedBox** [comment]
Hi @louisjasek 
The issue you raised (with mixing path separators) was resolved. However, I noticed that I was a little overzealous with replacing the Unix path separator with OS-dependent call - so I re-coded to ensure that datapackage.json always records as Unix path as mentioned in [spec](https://frictionlessdata.io/specs/data-resource/#url-or-path) - this now also is resolved.
I think a tidy-up of the bottom status bar, following the 'fix heading' issue was also warranted - so have gone ahead with this as well.

![image](https://user-images.githubusercontent.com/5482657/42544239-a8dd0a3a-84f3-11e8-8566-192cb1e6aa20.png)

![image](https://user-images.githubusercontent.com/5482657/42544186-34e91b78-84f3-11e8-8449-524e937254ba.png)
